### PR TITLE
[#148,149,190] Common code, Common config file, mqtt for blinds controller 

### PR DESCRIPTION
**Pull request by kuba8686**
- created common config file, instructions how to create at Wiki, .gitignore updated
- rearanging project structure, 
- all substations sketch putted at root/substations (every .ino files at folder with the same name!)
- every substations numbered starting from 1 (can be used as ID of station placed in JSON)
- all libraries (.h) putted at root/libraries
- to make Arduino IDE to see substations libs You have to set repo as a default Arduino catalog (at main view of Arduino IDE press Ctrl+comma)
- placing 3rd party libraries used at sketches (to make proper compiling for anybody who clones a repo)
- every substation with mqtt.h and init_config.h included at the top
- every substation checked for proper compilation
- adding MQTT communication for BlindsController (reciving and sending JSON), tested.

### CODE REVIEW

**Comment by kuba8686** [comment]
For easier review changes - filter the changes from commit 05c547c

**Comment by kuba8686** [comment]
- init_config file have include guard so redeclaration twice shouldnt be a problem,
- config file was included at all substation due to possibility that other communication method,  not only mqtt could be used,
and
- no more compliing error in Artur :) some undefined problem that dissapered 


> 
> 
> Due to init_config.h is included in each .ino files and MQTT.h I'm having compilation error. I think that including init_config.h in MQTT.h is enougch.

File: libraries/BlindsController/blinds.h [hunk 1]
```diff
@@ -138,5 +140,19 @@ static void blinds_move_targetValue(int targetValue)
         }
     }
 }
+
+/* Preparing JSON filled with station ID and currentValue */
+static String return_JSON(int id, int pos)
```
Comment (Siwy160) [inline libraries/BlindsController/blinds.h:145]: MQTT for blinds controller should be done as separate PR

**Comment by Siwy160** [comment]
I don't like that there are 4 ticket combined in one huge pull request.
Especially addition of external library,

**Comment by kuba8686** [comment]
> 
> 
> 02_LedControlerRGB:5:25: fatal error: init_config.h: No such file or directory
> #include "init_config.h"
> 
> I see that init_config is in .gitignore but there is no such file. I cannot compile. What should be in that file?



> 
> 
> 02_LedControlerRGB:5:25: fatal error: init_config.h: No such file or directory
> #include "init_config.h"
> 
> I see that init_config is in .gitignore but there is no such file. I cannot compile. What should be in that file?

common config file, instructions how to create at Wiki,
